File: android/src/main/kotlin/com/ottu/flutter/checkout/PaymentOptionsDisplaySettings.kt
package com.ottu.flutter.checkout

import kotlinx.serialization.Serializable
import kotlinx.serialization.json.JsonIgnoreUnknownKeys

@Serializable
@JsonIgnoreUnknownKeys
data class PaymentOptionsDisplaySettings(
    val mode: String,
    val defaultSelectedPgCode: String?,
    val visibleItemsCount: Int,
)

File: ios/Classes/PaymentOptionsDisplaySettings.swift
//
//  PaymentOptionsDisplaySettings.swift
//  ottu-flutter-checkout
//
//  Created by Vi on 03.11.2025.
//

struct PaymentOptionsDisplaySettings:Decodable {
    let mode: String
    let defaultSelectedPgCode: String?
    let visibleItemsCount: Int
}
